### ISSUE TITLE
 Unable to bind F13-F24, detected as an invalid keybind

### ISSUE THREAD

**Issue by viperfan7** [marcincichocki/breach-protocol-autosolver#346]
**Describe the bug**

Unable to bind F13-24,

Gives error:
```
[Window Title]
Breach Protocol Autosolver

[Main Instruction]
Invalid key bind

[Content]
Key bind can contain multiple modifiers and a single key code.

[Ok]
```
**To Reproduce**

Attempt to bind F13-F24

**Expected behavior**

Accepts it as a valid input

**Software information**

- Operating System: Windows 10
- _Cyberpunk 2077_ platform: Steam
- _BPA_ version: 2.8.3
- relevant _BPA_ settings: All input settings

**Additional files**

It does properly detect the key, after the key is released it gives the error

**Comment by marcincichocki** [comment]
F13 to F19 should work, I validated that this issue occurs with F20-F24 keys.

**Comment by marcincichocki** [comment]
Fix is deployed in [v2.8.5](https://github.com/marcincichocki/breach-protocol-autosolver/releases/tag/v2.8.5)
